.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: cpp
```

Comment:
Please use 4-space indentation in this file.

Reply:
done